03/MenuItem.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+class MenuItem extends React.Component {
+ render() {
+     const {text, url} = this.props
+ return (
+    <li> 
+        <a href = {url} > {text} </a>
```

Comment:
Raczej nie używa się takich odstępów, jak już to: `<a href={ url }>{ text }</a>`